molgenis-data-mapper/src/main/java/org/molgenis/data/mapper/service/impl/MappingServiceImpl.java

```diff
@@ -46,6 +55,15 @@
 	private MappingProjectRepository mappingProjectRepository;
 
 	@Autowired
+	private MappingTargetRepository mappingTargetRepository;
+
+	@Autowired
+	private EntityMappingRepository entityMappingRepository;
+
+	@Autowired
+	private AttributeMappingRepository attributeMappingRepository;
+
+	@Autowired
```

Comment:
The idea was to hide this implementation from the service and let the mappingprojectrepo delegate the incoming work.